.goreleaser.yml

```diff
@@ -0,0 +1,16 @@
+# .goreleaser.yml
+# Build customization
+builds:
+  - main: ./cmd/cs/
+    binary: cs
+    goos:
+      - windows
+      - darwin
+      - linux
+    goarch:
+      - amd64
+
+sign:
+  cmd: gpg
+  args: ["-a", "-u", "ops@exoscale.ch", "--detach-sign", "${artifact}"]
+  artifacts: all
```

Comment:
for the time being, `artifacts: none`

Reply:
okay and i have see your PR on goreleaser